content/06.know-your-problem.md

```diff
@@ -1,20 +1,28 @@
-All of these different designs have different assumptions and caveats, which cannot be ignored during a data analysis.
-Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated (for example, if all samples are from the same study site, if certain ethnic groups or zip codes are oversampled, if there are differences in processing dates or techniques.)
+Having a well-defined question, a hypothesis to test based on that question, and a clear analysis plan for proving the hypothesis are all crucial for carrying out a successful deep learning project.
+Just like it would be inadvisable to step foot in a laboratory and begin experiments without having a defined endpoint, a deep learning project should not be undertaken without preparation.
+Foremost, it is important to assess if a dataset exists that can prove or disprove the hypothesis, and obtaining said data and associated meta-data should be pursued as early on in the project as possible.
+A publication or resource might purportedly offer data that seems to be a good fit to test your hypothesis, but the act of obtaining the data can reveal numerous problems such as the data is unstructured when it is supposed to be structured, crucial meta-data such as sample stratification is missing, or the usable sample size is different than what is reported.
+Data collection should be documented or a data collection protocol should be created and specified in the project documentation.
+Information such as the resource used, the date downloaded, and the version of the dataset, if any, will help minimize operational confusion and will allow for transparency during the publication process.
+Once the data is obtained, it is easy to begin analyzing data without a good understanding of the study design, namely why the data were collected and how.
+Having appropriate meta-data and a comprehensive data dictionary is essential for any analysis, including one that involves deep learning.
+If at all possible, seek out a subject matter expert who has experience with this type of data.
+Receiving first-hand knowledge of the “gotchas" of a dataset will minimize the amount of guesswork and increase  the success rate of the deep learning project.
+For example, if the main reason why the data were collected was to test the impact of an intervention, then it may be the case that a randomized controlled trial was performed.
+However, if the standard of care requires an intervention, such as chemotherapy for cancer, then it would be unethical to randomize a patient into a placebo arm.
+Therefore, the design of the study may have been an observational one, either prospective or retrospective and may also incorporate some amount of matching.
+This matching means that cases and controls may have been selected so that the age range, gender, or weight distribution is similar.
```

Comment:
I would change the previous 2 sentences to: "Therefore, an observational study design is often considered, with the data either prospectively or retrospectively collected. In order to ensure similar distributions of important characteristics across study groups in the absence of randomization, individuals may be matched based on age, gender, or weight."
I do wonder if we're going too much down the epi route here. What is the study is an animal study or an in vitro study?

Reply:
Changed. 

I think the example is clear so I don't think it matters that it's not an in vitro study example.